lib/src/voip/models/call_backend.dart

```diff
@@ -0,0 +1,75 @@
+abstract class CallBackend {
```

Comment:
Is this an extension point we want external users to be able to use? Or should this be sealed? (The factory method wouldn't work if external users extend this)

Reply:
It is an API, that we define. It is easier to prohibit extending this class until we have a use case for that instead of prohibiting it later. And if we don't want to expose it externally, we could even thing about making the class private. We expose quite a big API for calls and we really should make explicit, what should be called externally and what not :D

Reply:
hm sealed classes can't seem to be extended over different files of the same library? considering how big the files are now we should then keep it abstract